### DISCUSSION TITLE
Add another button-row

### DISCUSSION THREAD

**Discussion by Glitchy-Tozier** [8VIM/8VIM#147] [Ideas]
Add one (or two) additional button-rows/columns at the borders of the keyboard.
This could facilitate a some things, like writing some special characters with dedicated buttons.

Important aspects of this idea:
- The rows can be moved and customized. This lets the user choose their position and also position the keyboard.
- The buttons can be assigned to any symbol the user wants.
- By default, they should be something like `.`, `,`, `!`, or `?` though.
- In the pictures I drew, there usually are about 4 buttons in a row. Maybe we should only use **3** though, to help with the concept of blind-typing.
- Here's a few images showing off the idea:

![8vim-drawing](https://user-images.githubusercontent.com/59611881/102660487-67038980-417b-11eb-8dc4-28b59af6c131.jpg)

![thisSolvesAllOurProblems](https://user-images.githubusercontent.com/59611881/102660475-61a63f00-417b-11eb-9de9-94ff2b5ca3d4.jpeg)

**Comment by flide** [comment]
This will make the keyboard a lot cluttered...
Moving this to discussion. This idea needs to be fleshed out a lot more before anything can be done for this.
